ui/raidboss/data/04-sb/raid/o4s.txt

```diff
@@ -3,12 +3,10 @@
 hideall "--Reset--"
 hideall "--sync--"
 
-0.0 "--Reset--" sync / 21:........:4000000F:/ window 100000 jump 0
-
 # -ii 23F2 23F8 2400 240C
 
 # Exfaust
-3.8 "--sync--" sync / 1[56]:[^:]*:Exdeath:23F2:/ window 3.8,0
+0.0 "--sync--" sync / 104:[^:]*:1($|:)/ window 0,1
```

Comment:
o4s has a door boss.  It uses the auto here to start the timeline for Exdeath vs Neo Exdeath.

Reply:
> o4s has a door boss. It uses the auto here to start the timeline for Exdeath vs Neo Exdeath. EDIT: I guess technically this will still work as the Neo auto will still resync the timeline. I just think it looks a little weird when the timeline is like "hey here's the door boss jk it's the final boss" and it'd be better to just not use a combat line here.

I was thinking the same as your edit - but yeah, I guess that will look a little awkward.

If we still want to avoid sync's/starts based on autos, and not have those few seconds of UI dystopia, we could use a combat line for both the door and final bosses, and shorten the sync on the second one so that it only looks back 200 seconds instead of all the way to 0.  And then add a jump to 800 on the AddedCombatant line for Neo Exdeath?  (Granted, I may be over-engineering this, but since the door boss issue is going to come up at least 6 more times before this PR is done, figured I'd propose it now...)

Reply:
I can go either way.  I think either of these approaches are good:
* just don't start the timeline until we know for sure
* start some "we're not sure" shared section with "Door Boss Ability?" "Final Boss Ability?" lines and then jump to the right one when we know.

Also, re: before this PR is done, for my own sake of reviewing this, once o4s is sorted out, can I land this one, and then the other ones be done in a separate PR?

Reply:
> I can go either way. I think either of these approaches are good:
> 
> * just don't start the timeline until we know for sure
> * start some "we're not sure" shared section with "Door Boss Ability?" "Final Boss Ability?" lines and then jump to the right one when we know.
> 
> Also, re: before this PR is done, for my own sake of reviewing this, once o4s is sorted out, can I land this one, and then the other ones be done in a separate PR?

I lean toward your first option.  I know we do the "not sure" stub elsewhere, but the autos occur early enough, and there's a long enough delay before real abilities start, that I think certainty is better than a stub.  I'll revert the o4s stuff and mark the PR as ready.